.github/workflows/downstream-project-spl.yml

```
@@ -0,0 +1,42 @@
+name: Downstream Project - SPL
+
+on:
+  push:
+    branches:
+      - master
+      - v[0-9]+.[0-9]+
+  pull_request:
+    branches:
+      - master
+      - v[0-9]+.[0-9]+
+    paths:
+      - "**.rs"
+      - "Cargo.toml"
+      - "Cargo.lock"
+      - "cargo-build-bpf"
+      - "cargo-test-bpf"
+      - "cargo-build-sbf"
+      - "cargo-test-sbf"
+      - "ci/downstream-projects/run-spl.sh"
+      - ".github/workflows/downstream-project.yml"
```

I think this is what you meant, correct?
```suggestion
      - ".github/workflows/downstream-project-spl.yml"
```

---

thank you! <3